config/prow/config_knative.yaml

```diff
@@ -31,24 +31,27 @@ presubmits:
           memory: 16Gi
     - unit-tests: true
       dot-dev: true
+      is-monitored: true
```

Comment:
Do you think `needs-monitor` would be a better name, as we already have `needs-dind`?

Reply:
make sense and done